Library/Formula/xalan-c.rb

```diff
@@ -30,6 +33,18 @@ def install
                             "--disable-silent-rules",
                             "--prefix=#{prefix}"
       system "make", "install"
+      if build.with? "docs"
+        ENV.prepend_path "PATH", "#{buildpath}/c/bin"
+        cd "xdocs" do
+          # Set the library path in the script which runs Xalan from
+          # the source tree, or else the libxalan-c.dylib won't be found.
+          inreplace "sources/make-xalan.sh", "\"${XALANCMD}\" \\",
+                    "export DYLD_FALLBACK_LIBRARY_PATH=#{buildpath}/c/lib:$DYLD_FALLBACK_LIBRARY_PATH\n\"${XALANCMD}\" \\"
+          system "./make-apiDocs.sh"
+        end
+        mkdir_p "#{prefix}/share/doc"
+        system "mv", "build/docs/xalan-c", "#{prefix}/share/doc"
```

Comment:
Thanks, I've updated it do use this.  There's so much hidden functionality in homebrew I don't know about!